equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileService.java

```diff
@@ -167,6 +170,21 @@ public <T extends CrmFormModelInterface> T convertCrmFileJsonToModel(JSONObject
         return (T) crmFormData;
     }
 
+
+    public void sectionsCheck(JSONObject crmFileJsonObject) {
+        JSONArray sectionsArray  = (JSONArray)((JSONObject)(crmFileJsonObject.getJSONObject("SectionStates"))).get("SectionName");
+        boolean hasCrm15Section =false;
+       if (sectionsArray != null) {
+            for (int i=0;i<sectionsArray.length();i++){
+                if (sectionsArray.getString(i).contains(CONTAINS_CRM15)){
```

Comment:
potential NPE, can you reverse the if?
`if (CONTAINS_CRM15.contains(sectionsArray.getString(i)))`

Reply:
done